.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - mountpoint-s3-[0-9]+.*
+      - mountpoint-s3-[a-z]+-[0-9]+.*
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: taiki-e/create-gh-release-action@v1
+      with:
+        prefix: mountpoint-s3(-[a-z]+)?
+        draft: true
+        # TODO: set it true after we have changlog template reviewed.
+        changelog: false
+        # TODO: set it false after we have changlog template reviewed.
+        allow-missing-changelog: true
+        branch: main
+        title: "Mountpoint for Amazon S3 $version"
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+  upload-assets:
+    name: Build and release on target ${{ matrix.runner.target}}
+    runs-on: ${{ matrix.runner.tags }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        runner:
+        - tags: [ubuntu-20.04] # GitHub-hosted
+          target: x86_64-unknown-linux-gnu
+        - tags: [self-hosted, linux, arm64] 
+          target: aarch64-unknown-linux-gnu
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Set up stable Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        override: true
+    - name: Install operating system dependencies
+      uses: ./.github/actions/install-dependencies
+      with:
+        fuseVersion: 2 
+    - name: Release mount-s3 binary
+      uses: taiki-e/upload-rust-binary-action@v1
+      with:
+        bin: mount-s3
+        token: ${{ secrets.GITHUB_TOKEN }}
+        checksum: sha512
+        asset: LICENSE
+
+
```

Comment:
release example: https://github.com/eslrahc-swa/mountpoint-s3/releases/tag/mountpoint-s3-0.0.1

client test:
```
# zyaoshen @ u40d82267bd7952 in ~/Downloads [13:03:11] 
$ ./mount-s3 jepsen-canary /tmp/mnt

# zyaoshen @ u40d82267bd7952 in ~/Downloads [13:03:22] 
$ cd /tmp/mnt 

# zyaoshen @ u40d82267bd7952 in /tmp/mnt [13:03:29] 
$ ls
0-src####                             16-src####                            20-src####                            27-src####  34-src####                            
```

Reply:
Where does the release message come from? I saw this from the example
```
mountpoint-s3-0.0.1

mountpoint-s3-0.0.1
```
Can we modify this text when doing a release?

Reply:
We can modify those before release, given the release is a draft, before we click the release button, team must double check all the artifacts is valid